mkosi/__init__.py

```diff
@@ -4738,9 +4739,20 @@ def run_build(args: Args, config: Config, *, resources: Path, package_dir: Optio
             remount += ["/usr"]
 
         for d in remount:
-            if Path(d).exists():
-                options = "ro" if d in ("/usr", "/opt") else "ro,nosuid,nodev,noexec"
-                run(["mount", "--rbind", d, d, "--options", options])
```

Comment:
Please just call `.resolve()` on the source path here, that should be sufficient to fix the issue.